server/polar/web_backoffice/organizations/endpoints.py

```diff
@@ -707,6 +708,130 @@ async def remove_member(
         )
 
 
+@router.api_route(
+    "/{id}/setup_manual_payout",
+    name="organizations:setup_manual_payout",
+    methods=["GET", "POST"],
+)
+async def setup_manual_payout(
+    request: Request,
+    id: UUID4,
+    session: AsyncSession = Depends(get_db_session),
+) -> Any:
+    org_repo = OrganizationRepository.from_session(session)
+    organization = await org_repo.get_by_id(id)
+    if not organization:
+        raise HTTPException(status_code=404)
+
+    # Check if organization already has an account
+    if organization.account_id is not None:
+        raise HTTPException(
+            status_code=400, detail="Organization already has an account"
+        )
+
+    if request.method == "POST":
+        # Create a manual account for the organization
+        # Get country from form data
+        data = await request.form()
+        country = data.get("country", "US")
+
+        # Get the first user of the organization as admin
+        user_repository = UserRepository.from_session(session)
+        users = await user_repository.get_all_by_organization(organization.id)
+        if not users:
+            raise HTTPException(status_code=400, detail="Organization has no users")
+
+        admin_user = users[0]  # Use first user as admin
+
+        # Create manual account
+        account = Account(
+            id=generate_uuid(),
+            account_type=AccountType.manual,
+            admin_id=admin_user.id,
+            country=country,
+            currency="usd",
+            is_details_submitted=True,
+            is_charges_enabled=True,
+            is_payouts_enabled=True,
+            processor_fees_applicable=False,  # No processor fees for manual accounts
+            status=Account.Status.ACTIVE,
+        )
+
+        session.add(account)
+        await session.flush()
+
+        # Associate account with organization
+        organization = await org_repo.update(
+            organization, update_dict={"account_id": account.id}
+        )
```

Comment:
Could be nice to move all the account creation logic inside `account/service.py`.